### PR TITLE
[ticket/15900] Converting install/index.html to UTF-8 and fixing tag

### PR DESCRIPTION
**Pull request by orynider**
PHPBB3-15900

Checklist:

- [ ] Correct branch: master for new features; 3.2.x for fixes
- [ ] Tests pass
- [ ] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [ ] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.2.x/development/git.html)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-15900

### CODE REVIEW

**Comment by 3D-I** [comment]
There are 22 other occurrences inside phpBB.

As has already been said, I don't see how useful it is to change this as for example my editor doesn't display the syntax as wrong. It's HTML4 and if you have the corresponding syntax highlighter in your editor you'll see that there's no problem.

immagine

**Comment by orynider** [comment]
The color is same for charset and for his value.

**Comment by dark-1** [comment]
> PHPBB3-15900
> ...
> Tracker ticket (set the ticket ID to **your ticket ID**):
> 
> https://tracker.phpbb.com/browse/PHPBB3-12345

You forgot to change the above part to:
> https://tracker.phpbb.com/browse/PHPBB3-15900


Also this commit has no use for whatever good reason you are making a PR.
It should be something of helpful.

Best regards 👍

**Comment by orynider** [comment]
Editing comment using js-comment-edit-button is not working from were I'm online atm.

**Comment by orynider** [comment]
> 
> 
> Editing comment using js-comment-edit-button is not working from were I'm online atm.

I'm using a ff using agent layer and now works.

File: phpBB/install/index.html [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
+<meta http-equiv="Content-Type" content="text/html" charset="UTF-8" />
 <meta http-equiv="refresh" content="0; url=./app.php" />
 </head>
 
 <body bgcolor="#FFFFFF" text="#000000">
 
 </body>
-</html>
+</html>
```
Comment (3D-I) [inline phpBB/install/index.html]: Add a new blank line to the end of the file please.

File: phpBB/install/index.html [hunk 2]
```diff
@@ -1,11 +1,11 @@
 <html>
 <head>
 <title></title>
-<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
+<meta http-equiv="Content-Type" content="text/html" charset="UTF-8" />
```
Comment (3D-I) [inline phpBB/install/index.html]: The original line is correct HTML 4.0.1, if you want to change it to UTF-8 pls do

<meta http-equiv="content-type" content="text/html; charset=UTF-8">

Instead if you want to port it to HTML5 and UTF-8 pls do

`<meta charset="UTF-8">`

Pls note there are at leat 22 other occurencies in phpBB.

**Comment by hanakin** [comment]
This is rather pointless and is something that can easily and more properly be fixed universally once we finish the new front-end

**Comment by marc1706** [comment]
Closing as previously explained.

**Pull request closed without merging**